### PR TITLE
Korean Update

### PR DESCRIPTION
**Pull request by VenusGirl**
The sentence that foreigners misinterpreted Korean as an exercise was originally corrected by Koreans according to Korean spelling and grammar.

### CODE REVIEW

File: src/OfficeToolPlus/Dictionaries/Languages/ko-kr.xaml [hunk 1]
```diff
@@ -691,9 +691,10 @@
     <!-- Example: Creating Office ISO file, file size: 3.1GB, 45% completed. -->
     <sys:String x:Key="Task_Creating_ISO">Office ISO 파일 만들기, 파일 크기: {0}, {1} 완료되었습니다.</sys:String>
     <sys:String x:Key="Task_ODT">지금 배포 중, 잠시 기다려 주십시오.</sys:String>
-    <sys:String x:Key="Task_SPP">SPP 실행 중, 잠시 가다려 주십시오.</sys:String>
-    <sys:String x:Key="Task_Searching_KMS_Events">KMS 정품인정 이벤트를 검색 중, 잠시 기다려 주십시오.</sys:String>
-    <sys:String x:Key="Task_Searching_IAF_Events">오라인 정품인정 이벤트를 검색 중, 잠시 기다려 주십시오.</sys:String>
+    <sys:String x:Key="Task_OSPP">OSPP가 작업 중, 잠시 기다려 주십시오.</sys:String>
```
Comment (YerongAI) [inline src/OfficeToolPlus/Dictionaries/Languages/ko-kr.xaml:694]: The OSPP was removed.

File: src/OfficeToolPlus/Dictionaries/Languages/ko-kr.xaml [hunk 2]
```diff
@@ -727,17 +728,16 @@
     <sys:String x:Key="Ques_Delete_Installation">{0}을(를) 삭제하시겠습니까?</sys:String>
     <sys:String x:Key="Ques_Delete_Installation_Languages">{0}을(를) 삭제하시겠습니까?</sys:String>
     <sys:String x:Key="Ques_Install_iSlide">지금 iSlide를 설치하시겠습니까?</sys:String>
-    <sys:String x:Key="Ques_Clear_Activation">모든 라이선스 및 키를 제거하시겠습니까? 이 작업을 수행한 후에는 라이선스를 다시 설치하고 Office를 다시 정품 인증해야 합니다.</sys:String>
-    <sys:String x:Key="Ques_Install_Licenses">Office 라이선스를 설치한 후 Office에 정품 인증 정보가 표시되지 않을 수 있지만 Office Tool Plus를 통해 정보를 보거나 Office를 정품 인증할 수 있습니다. 계속하시겠습니까?</sys:String>
+    <sys:String x:Key="Ques_Clear_Activation">모든 라이선스 및 키를 제거하시겠습니까? 이 작업을 수행한 후에는 라이선스를 다시 설치하고 Office를 다시 활성화해야 합니다.</sys:String>
+    <sys:String x:Key="Ques_Install_Licenses">Office 라이선스를 설치한 후 Office에 활성화 정보가 표시되지 않을 수 있지만 Office Tool Plus를 통해 정보를 보거나 Office를 활성화할 수 있습니다. 계속하시겠습니까?</sys:String>
     <sys:String x:Key="Ques_Uninstall_Licenses">모든 Office 라이선스를 제거하시겠습니까?</sys:String>
     <sys:String x:Key="Ques_Uninstall_All_Keys">모든 Office 제품 키를 제거하시겠습니까? 필요한 경우 키를 먼저 백업하십시오.</sys:String>
-    <sys:String x:Key="Ques_Uninstall_Product_Key">이 제품 키를 제거하겠습니까?</sys:String>
```
Comment (YerongAI) [inline src/OfficeToolPlus/Dictionaries/Languages/ko-kr.xaml:734]: The `Ques_Uninstall_Product_Key` is missing.

**Review by YerongAI — CHANGES_REQUESTED** [review]

**Comment by yaeyaya** [comment]
I think VenusGirl's translation is better cause it's original Korean, not the textbook-style or dictionary-style from me. 😆

**Comment by yaeyaya** [comment]
@VenusGirl 님, 번역 내용을 수정하고 개선해서 감사해요. 저도 더 어울리는 번역을 배웠어요. ㅋㅋ

**Review by YerongAI — APPROVED** [review]

**Pull request merged**